engine/Cargo.toml

```diff
@@ -45,7 +45,7 @@ indexmap = { version = "2.1.0", features = ["serde"] }
 indoc = "2.0.1"
 regex = "1.10.4"
 serde_json = { version = "1", features = ["float_roundtrip", "preserve_order"] }
-serde = { version = "1", features = ["derive"] }
+serde = { version = "1", features = ["derive", "rc"] }
```

Comment:
whats rc?

Reply:
Rc is a feature that we use to serialize Arc types.